auth0/resource_auth0_organization.go

```diff
@@ -210,7 +153,53 @@ func deleteOrganization(ctx context.Context, d *schema.ResourceData, m interface
 	return nil
 }
 
-func buildOrganization(d *schema.ResourceData) *management.Organization {
+func updateOrganizationConnections(d *schema.ResourceData, api *management.Management) error {
+	toAdd, toRemove := Diff(d, "connections")
+
+	var err error
+	toRemove.Elem(func(data ResourceData) {
```

Comment:
After we updated the schema we were getting at times weird errors about the connection already being there (409s).

This was because if we change the bool flag on a connection we get the following diff:

```
 - connections {
    - assign_membership_on_login = false -> null
    - connection_id              = "con_E68qtlsIB68K6Yz6" -> null
  }
  + connections {
    + assign_membership_on_login = true
    + connection_id              = "con_E68qtlsIB68K6Yz6"
  }
```

So to correctly update the connection without adding complicated diffing logic, we delete the connection and then re-add it.